src/marias3.c

```diff
@@ -167,8 +168,8 @@ const char *ms3_error(uint8_t errcode)
 uint8_t ms3_list(ms3_st *ms3, const char *bucket, const char *prefix,
                  ms3_list_st **list)
 {
-  (void) prefix;
   uint8_t res = 0;
+  (void) prefix;
```

Comment:
This line can probably actually die anyway. It was there for the early betas, before prefix was supported.